tiles/src/main/java/com/protomaps/basemap/layers/Housenumbers.java

```diff
@@ -0,0 +1,34 @@
+package com.protomaps.basemap.layers;
+
+import com.onthegomap.planetiler.FeatureCollector;
+import com.onthegomap.planetiler.ForwardingProfile;
+import com.onthegomap.planetiler.VectorTile;
+import com.onthegomap.planetiler.geo.GeometryException;
+import com.onthegomap.planetiler.reader.SourceFeature;
+import com.protomaps.basemap.feature.FeatureId;
+
+import java.util.List;
+
+public class Housenumbers  implements ForwardingProfile.LayerPostProcesser {
+  public void processOsm(SourceFeature sf, FeatureCollector features) {
+    if (sf.hasTag("addr:housenumber")) {
+      if (sf.isPoint()) {
+        features.point(this.name())
+          .setId(FeatureId.create(sf))
+          .setAttr("housenumber", sf.getString("addr:housenumber"))
+          .setZoomRange(10, 17);
```

Comment:
We should limit the house numbers to exist only in the zoom 15 tiles, in order to not bloat the tile size. A visual reference we use for deciding what data goes in what zoom is http://tangrams.github.io/bubble-wrap - house numbers only appear above zoom ~18 which means we can put them in our tileset at 15 and define the style to appear > 18, etc. Is that zoom level good enough for your use case?

Reply:
Yes, I think so :)